test/gtest/uct/ib/test_ib.cc

```diff
@@ -452,7 +452,9 @@ UCS_TEST_P(test_uct_ib_sl, check_ib_sl_config) {
         max_avail_sl = UCT_IB_SL_NUM;
     } else {
         max_avail_sl = (uint8_t)strtoul(max_avail_sl_str, NULL, 0);
-        ASSERT_LE(max_avail_sl, UCT_IB_SL_NUM);
+        if (max_avail_sl > UCT_IB_SL_NUM) {
+            max_avail_sl = UCT_IB_SL_NUM;
+        }
```

Comment:
Before squashing I had to solve a coverity issue that says we should make sure that `max_avail_sl`'s value is not out of bounds.
@gleon99 / @yosefe  Do you know why `ASSERT_LE` is not enough here?

Reply:
i guess it's because of casting to uint8_t, that is before the ASSERT_LE; i gues we can do something like
`max_avail_sl = std::min(strtoul(max_avail_sl_str, NULL, 0), UCT_IB_SL_NUM)`